### ISSUE TITLE
Recommended way of click and hover actions for each part

### ISSUE THREAD

**Issue by dominikfeininger** [greghub/funnel-graph-js#33]
Hi, 

- I would like to add some hover action to display more data
- click on a section/ part and do eg. navigation. Therefore I need to access the data.

Whats your recommended way to do so? 

Thx.

**Comment by ginow** [comment]
yes need this feature so that on click of any section of the funnel we can navigate to data
